nodejs/README.md

```diff
@@ -0,0 +1,208 @@
+# Node.js
+
+> [Node.js](https://nodejs.org/) is a JavaScript runtime built on Chrome's V8
+> JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that
+> makes it lightweight and efficient. Node.js' package ecosystem, npm, is the
+> largest ecosystem of open source libraries in the world.
+
+This is a living guide that describes some best practices around using Node.js
+at 18F. For areas we don't have a specific recommendation, we try to document
+what folks are currently using to help inform your decision.
+
+This guide will also help facilitate discussions around these technologies so we
+can identify where there is consensus and where there is disagreement.
+
+For each area, we'll include Recommendations, Suggestions, or Considerations.
+Recommendations are practices where there is a strong consensus on the team.
+Suggestions are practices where there is some agreement but many reasons why you
+might choose something different. Considerations are tips to help you decide for
+yourself what is right for your project.
+
+If you have any questions, hop into the
+[#javascript](https://gsa-tts.slack.com/messages/javascript/) Slack channel.
+
+
+## Twelve-factor web applications
```

Comment:
@jessieay I'd say that's a great idea. The cloud.gov team has been talking about making something similar, with prospective and new cloud.gov customers as the intended audience... let's repurpose in one direction or the other.